tox.ini

```diff
@@ -2,6 +2,8 @@
 envlist =
     docs, packaging, lint-py2, lint-py3,
     py27, py33, py34, py35, py36, py37, pypy
+_pip = python {toxinidir}/.tox_helpers/runpip.py
```

Comment:
A `[helpers]` section with a nice comment documenting why all these things are done this way would be nice! ^.^